packages/core/state/metadata/logics.ts

```diff
@@ -20,14 +21,24 @@ const requestAnnotations = createLogic({
         const { getState, httpClient } = deps;
         const applicationVersion = interaction.selectors.getApplicationVersion(getState());
         const baseUrl = interaction.selectors.getFileExplorerServiceBaseUrl(getState());
+        const displayAnnotations = selection.selectors.getAnnotationsToDisplay(getState());
         const annotationService = new AnnotationService({
             applicationVersion,
             baseUrl,
             httpClient,
         });
 
+        let annotations: Annotation[] = [];
+
         try {
-            const annotations = await annotationService.fetchAnnotations();
+            annotations = await annotationService.fetchAnnotations();
+        } catch (err) {
+            console.error("Failed to fetch annotations", err);
+        }
+
+        dispatch(receiveAnnotations(annotations));
```

Comment:
We don't want to dispatch this if we don't have `annotations` & the defaults wouldn't be able to render anyway without them. In that case it seems like it would be best to have the catch statement look like:
```
            console.error("Failed to fetch annotations", err);
            done();
            return;
```

and the `dispatch(receiveAnnotations(annotations));` should move into the try block as the last line in it.

Reply:
resolved in [f3b21ab](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/75/commits/f3b21abbf790b9287163be67ba25e771407927e0)